### ISSUE TITLE
Problem z akcją https://api.allegro.pl.allegrosandbox.pl/sale/product-offers

### ISSUE THREAD

**Issue by MaciejFrackowiak** [allegro/allegro-api#10506]
### Discussed in https://github.com/allegro/allegro-api/discussions/10505

<div type='discussions-op-text'>

<sup>Originally posted by **ryszardlet** December 13, 2024</sup>
Czy usunięte zostały pola: endingAt i endedBy? </div>

**Comment by MaciejFrackowiak** [comment]
@ryszardlet Nie, te pola są dostępne w odpowiedzi do [POST /sale/product-offers](https://developer.allegro.pl/documentation/#operation/createProductOffers) i [PATCH /sale/product-offers/{offerId}](https://developer.allegro.pl/documentation/#operation/editProductOffers).

**Comment by ryszardlet** [comment]
Dziwne, dostałem taką odpowiedź (a3f9a40cbc7b2163):
`{"errors":[{"code":"UnknownJSONProperty","message":"Unknown properties found in the request.","details":null,"path":null,"userMessage":"Przesłane dane zawierają nieobsługiwane właściwości. Skontaktuj się z dostawcą oprogramowania.","metadata":{"unknownProperties":"...endingAt, ...endedBy"}}]}`

**Comment by MaciejFrackowiak** [comment]
I dostałeś prawidłową odpowiedź - nie są to pola, które powinny zostać przekazane w requeście. Zwracamy je wyłącznie w odpowiedzi do wykonanego requestu.

**Comment by ryszardlet** [comment]
Wszystko przez to, że musiałem poprawić przesyłanie ofert (problem ze swaggerem - kiedyś to chyba omawialiśmy już). Najwyraźniej za dużo pól przeniosłem.
